platform/platform.json

```diff
@@ -1,11 +1,11 @@
 {
-  "version": "2.4.1",
-  "mobileAppVersion": "2.4.1",
-  "releaseNotes": "* SendBird Chat extension\n* Advertising extension\n* Bugfixes and stability improvements",
+  "version": "2.4.2",
+  "mobileAppVersion": "2.4.2",
+  "releaseNotes": "* Added helper scripts for build information\n* Bugfixes and stability improvements",
```

Comment:
let's go for more descriptive notes. Mention push notifications , firebase , iOS14 , desugarring and stuff